src/Generator/Generators/CSharp/CSharpTextTemplate.cs

```diff
@@ -2581,6 +2581,17 @@ public struct ParamMarshal
             return paramMarshal;
         }
 
+        private CSharpTypePrinterResult ReturnTypeCheckIsParamRefConvertible(Parameter param)
+        {
+            if(!param.HasDefaultValue && !param.IsOut && !param.IsInOut
```

Comment:
@ddobrev I tried to copy the HandleDeafultValuesPass to a new pass but adding that to any test project failed the build.
I also tried adding the existing pass to the passes in both Basic.cs and CSharpTemp.cs but both times the build failed without any specific error messages.
Am I missing something? How is it supposed to generate overloads? I mean how to call it to generate overloads for default values?